packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -157,9 +157,12 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
         remoteVideoTileMenuOptions={remoteVideoTileMenuOptions}
         /* @conditional-compile-remove(vertical-gallery) */
         overflowGalleryPosition={overflowGalleryPosition}
-        /* @conditional-compile-remove(rooms) */
+        /* @conditional-compile-remove(click-to-call) */
```

Comment:
I think all 3 features should be part of this API removal and whichever goes first releases the prop on the video gallery. this cc wont work if c2c doesn't go before the other two